src/internal/connector/sharepoint/data_collections.go

```diff
@@ -30,6 +30,7 @@ func DataCollections(
 	ctx context.Context,
 	itemClient *http.Client,
 	selector selectors.Selector,
+	metadata []data.RestoreCollection,
```

Comment:
not needed yet
```suggestion
```

Reply:
Resolved with  [05a83de](https://github.com/alcionai/corso/pull/2528/commits/05a83de91a67c9bcac79a1db1c017a1e5611d20d)